modules/mod_admin/mod_admin.erl

```diff
@@ -24,6 +24,7 @@
 -mod_description("Provides administrative interface for editing pages, media, users etc.").
 -mod_depends([base, authentication, mod_search, mod_mqtt]).
 -mod_provides([admin]).
+-mod_depends([bootstrap]).
```

Comment:
Does this mean users will have to add `mod_bootstrap` to their `config`?

Reply:
No, just any module that provides `bootstrap`, which might also be their own module.